ASM/ootSymbols.ld

```diff
@@ -21,3 +21,16 @@ z64_sEquipAnimTimer = 0x8039EAB4;
 z64_sEquipMoveTimer = 0x8039EAB8;
 gActorOverlayTable = 0x800E8530;
 Message_CloseTextbox = 0x800d6218;
+Audio_PlaySfxGeneral = 0x800C806C;
+Play_SaveSceneFlags = 0x8009D894;
+Sram_WriteSave_call = 0x800905D4;
+Interface_SetDoAction = 0x80071E6C;
+Interface_ChangeHudVisibilityMode = 0x8006D8E0;
+func_800F64E0 = 0x800C7200;
+gRegEditor = 0x8011BA00;
+gSaveContext = 0x8011A5D0;
```

Comment:
We already have a definition for this. If you're going to change to doing it this way, it should be done for all of the existing usages too.